cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -360,64 +352,56 @@ private void generateHtmlReport(Project project, PrintStream out, TestReport tes
         }
     }
 
-    private int runTestSuit(Path moduleTestCache, Target target, Collection<Path> testDependencies,
-                            Module module) throws IOException, InterruptedException {
+    private int runTestSuit(Path testCachePath, Target target, String packageName, String orgName) throws IOException,
+            InterruptedException {
         List<String> cmdArgs = new ArrayList<>();
         cmdArgs.add(System.getProperty("java.command"));
         String mainClassName = TesterinaConstants.TESTERINA_LAUNCHER_CLASS_NAME;
-        String orgName = module.packageInstance().packageOrg().toString();
-        String packageName = module.packageInstance().packageName().toString();
-        String moduleName = module.isDefaultModule() ? "" : module.moduleName().moduleNamePart();
-
-        String jacocoAgentJarPath = Paths.get(System.getProperty(BALLERINA_HOME)).resolve(BALLERINA_HOME_BRE)
-                .resolve(BALLERINA_HOME_LIB).resolve(TesterinaConstants.AGENT_FILE_NAME).toString();
-
 
         if (coverage) {
+            String jacocoAgentJarPath = Paths.get(System.getProperty(BALLERINA_HOME)).resolve(BALLERINA_HOME_BRE)
+                    .resolve(BALLERINA_HOME_LIB).resolve(TesterinaConstants.AGENT_FILE_NAME).toString();
             String agentCommand = "-javaagent:"
                     + jacocoAgentJarPath
                     + "=destfile="
                     + target.getTestsCachePath().resolve(TesterinaConstants.COVERAGE_DIR)
                     .resolve(TesterinaConstants.EXEC_FILE_NAME).toString();
+
             if (!TesterinaConstants.DOT.equals(packageName) && this.includesInCoverage == null) {
                 // add user defined classes for generating the jacoco exec file
                 agentCommand += ",includes=" + orgName + ".*";
             } else {
                 agentCommand += ",includes=" + this.includesInCoverage;
             }
+
             cmdArgs.add(agentCommand);
         }
 
-        String classPath = getClassPath(testDependencies);
-        cmdArgs.addAll(Lists.of("-cp", classPath));
+        cmdArgs.addAll(Lists.of("-cp", getClassPath()));
+
         if (isInDebugMode()) {
             cmdArgs.add(getDebugArgs(this.err));
         }
         cmdArgs.add(mainClassName);
-        cmdArgs.add(moduleTestCache.toString());
-        cmdArgs.addAll(args);
-        cmdArgs.add(target.path().toString());
-        cmdArgs.add(orgName);
-        cmdArgs.add(packageName);
-        cmdArgs.add("\"" + moduleName + "\""); // see JDK-7028124
+
+        cmdArgs.add(testCachePath.toString());      // 0
```

Comment:
Let's describe what is done here in a single meaningful comment, rather than these comments at the EOL.

Reply:
removed comments and added a description